### ISSUE TITLE
Package to be used with qt-pods

### ISSUE THREAD

**Issue by cybercatalyst** [xiangzhai/pictureflow-qt#2]
Really cool. Are you maintaining the project? At least gh shows that you worked on it for the most recent time.

**Comment by xiangzhai** [comment]
I only migrated it to Qt 5.2.x, that is easy! but if there are some bugs, please ping me ;)

**Comment by cybercatalyst** [comment]
Would you be able to package it for qt-pods?

**Comment by xiangzhai** [comment]
Hi @cybercatalyst https://github.com/xiangzhai/pictureflow-qt/commit/b6011c6dcecc166ac4f87b70e262b4794d2bd934 is it OK for pod rules?

**Comment by cybercatalyst** [comment]
Not quite, I will help you. Just hang on, because I have to work now, but I will definately send you a pr again. Thanks in advance
